microk8s-resources/wrappers/addon-lists.yaml

```diff
@@ -59,9 +59,10 @@ microk8s-addons:
 
     - name: "istio"
       description: "Core Istio service mesh services"
-      version: "1.5.1"
+      version: "1.9.4"
       check_status: "pod/istio"
       supported_architectures:
+        - arm64
```

Comment:
ARM support is only for istioctl client libraries, not the in cluster components

Reply:
What components aren't supporting ARM64. Thought it was Envoy that was the blocker.

Reply:
All of them. We have a custom build of envoy.

There is nothing major blocking support other than someone modifying the build+testing infra, but its not been done

Reply:
ok i will change the pr to draft. Is there a issue/s to follow to track this progress?

Reply:
It would be good to merge the 1.9 upgrade part but not the ARM part. https://github.com/istio/istio/issues/21094 tracks this

Reply:
ok removed the arm64 parts. Will rename pr.